index.html

```diff
@@ -3,14 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="style.css" />
-    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700|Old+Standard+TT&display=swap&subset=cyrillic" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700|Old+Standard+TT&display=swap&subset=cyrillic"
+      rel="stylesheet"
+    />
     <title>Портфолио Junior верстальщика</title>
   </head>
 
   <body>
     <header class="page-header">
       <div class="container">
-        Judie's home
+        <a class="header-email" href="katarina.santalova@gmail.com"
+          >katarina.santalova@gmail.com</a
+        >
+        <a class="header-phone" href="+7 964 794-53-65">+7 964 794-53-65</a>
```

Comment:
href should include tel: prefix and should not include whitespaces and hyphens

Reply:
fixed